config/dev.exs

```diff
@@ -11,8 +11,7 @@ config :livebook, LivebookWeb.Endpoint,
   http: [
     ip: {127, 0, 0, 1},
     port: 4000,
-    http_1_options: [max_header_length: 32768],
-    http_2_options: [max_header_value_length: 32768]
+    http_1_options: [max_header_length: 32768]
```

Comment:
For http2 `max_header_value_length` has been removed, now there is a different option:

```
  * `max_header_block_size`: The maximum permitted length of a field block of an HTTP/2 request
    (expressed as the number of compressed bytes). Includes any concatenated block fragments from
    continuation frames. Defaults to 50_000 bytes
```

And we should be good with the default value?